GuessingGame.Tests/UnitTest1.cs

```diff
@@ -7,16 +7,33 @@ namespace GuessingGame.Tests;
 public class UnitTest1
 {
     [Fact]
-    public void GetResult_found()
+    public void Test_For_Winning_GuessingGameResult()
     {
 
-        int minRange = 4;
+        int minRange = 1;
         int maxRange = 5;
         int guessNumber = 3;
         INumberGenerator numberGenerator = Substitute.For<INumberGenerator>();
         numberGenerator.Generate(minRange, maxRange).Returns(guessNumber);
-        Service service = new Service(numberGenerator);
-        string result=service.GetResult(minRange,maxRange,6);
+        GuessingGameService service = new GuessingGameService(numberGenerator);
+        string result = service.GuessingGameResult(minRange, maxRange, 5);
+        Assert.Equal("Congratulations", result);

```

Comment:
Have you tried running these tests? I don't think it works because it gets stuck and Console.ReadLine(). I think you have too much coupling of your methods. Consider refactoring your methods so they are easier to test.

Reply:
Check Now . Test Case is executing now